### PR TITLE
Fixing column header for JSON to CSV encoding

### PR DESCRIPTION
**Pull request by sukaant**
smartcitiesdata/discovery_ui#195

### CODE REVIEW

File: lib/discovery_api_web/views/data_view.ex [hunk 1]
```diff
@@ -86,4 +84,12 @@ defmodule DiscoveryApiWeb.DataView do
     |> Map.get("feature")
     |> Jason.decode!()
   end
+
+  defp do_transform(el, false) do
```
Comment (ManApart) [inline lib/discovery_api_web/views/data_view.ex]: Could we rename el to element or column or something? (Same for both do_transforms)
```suggestion
  defp do_transform(element, false) do
```

**Comment by ManApart** [comment]
Your build failed due to a timeout for a presto integration test. We've had the same happen and a rebuild fixed it. I took the liberty of running rebuild on your PR.
We probably should up the timeout for that test at some point.

File: lib/discovery_api_web/views/data_view.ex (hunk 1 above)
Comment (sukaant) [reply lib/discovery_api_web/views/data_view.ex]: sure

Comment (abrock-accenture) [reply lib/discovery_api_web/views/data_view.ex]: 👍

**Pull request merged**